ale_linters/proto/protoc_gen_lint.vim

```diff
@@ -0,0 +1,22 @@
+" Author: Jeff Willette <jrwillette88@gmail.com>
+" Description: run the protoc-gen-lint plugin for the protoc binary
+
+function! ale_linters#proto#protoc_gen_lint#GetCommand(buffer) abort
+    let l:dirname = expand('#' . a:buffer . ':p:h')
+    let l:filename = expand('#' . a:buffer)
+
+    "\   'command': 'protoc -I $(dirname %s) --lint_out=. %s',
+
+    return 'protoc'
+    \   . ' -I ' . ale#Escape(l:dirname)
+    \   . ' --lint_out=. ' . ale#Escape(l:filename)
```

Comment:
You can use just `%s` on the end there. ALE will automatically replace that with the filename.